engine/src/main/scala/cromwell/webservice/metadata/MetadataBuilderActor.scala

```diff
@@ -232,7 +232,7 @@ class MetadataBuilderActor(serviceRegistryActor: ActorRef) extends LoggingFSM[Me
       allDone
     case Event(WorkflowOutputsResponse(id, events), _) =>
       // Add in an empty output event if there aren't already any output events.
-      val hasOutputs = events collectFirst { case MetadataEvent(MetadataKey(_, _, key), _, _) if key == WorkflowMetadataKeys.Outputs => () } isDefined
+      val hasOutputs = events collectFirst { case MetadataEvent(MetadataKey(_, _, key), _, _) if key.startsWith(WorkflowMetadataKeys.Outputs + ":") => () } isDefined
```

Comment:
Is this rather odd combination of `collectFirst`, `case x if y => ()` and `isDefined` just equivalent to `exists`?